p2p/src/lib.rs

```diff
@@ -251,6 +268,8 @@ where
 
     /// Update the list of peers we know about or update a known peers list of addresses
     fn peer_discovered(&mut self, peers: &[net::AddrInfo<NetworkingBackend>]) -> error::Result<()> {
+        log::debug!("discovered {} new peers", peers.len());
```

Comment:
If this doesn't happen too often, it's probably worth making it info. Bitcoin does.